package.json

```diff
@@ -25,15 +25,15 @@
     "@bcrs-shared-components/corp-type-module": "1.0.16",
     "@bcrs-shared-components/correct-name": "1.0.46",
     "@bcrs-shared-components/court-order-poa": "3.0.11",
-    "@bcrs-shared-components/date-picker": "1.3.0",
+    "@bcrs-shared-components/date-picker": "1.3.1",
     "@bcrs-shared-components/document-delivery": "1.2.0",
     "@bcrs-shared-components/effective-date-time": "1.1.15",
     "@bcrs-shared-components/enums": "1.1.10",
     "@bcrs-shared-components/expandable-help": "1.0.1",
     "@bcrs-shared-components/genesys-web-message": "1.0.0",
     "@bcrs-shared-components/help-business-number": "1.1.1",
     "@bcrs-shared-components/interfaces": "1.1.15",
-    "@bcrs-shared-components/jurisdiction": "1.1.3",
+    "@bcrs-shared-components/jurisdiction": "1.1.4",
```

Comment:
I added prop `showAppendIcon` to allow hiding the icon (so I can overlay an 'X' icon there instead).